crates/swift-bridge-build/src/package.rs

```diff
@@ -0,0 +1,287 @@
+//! Generate a Swift Package from Rust code
+
+use std::collections::HashMap;
+use std::fs;
+use std::fs::OpenOptions;
+use std::io::Write;
+use std::path::Path;
+use std::process::Command;
+
+/// Config for generating Swift packages
+pub struct GeneratePackageConfig<'a> {
```

Comment:
Minor note:

How do you feel about using the word `create` instead of generate?

i.e. `CreatePackageConfig` .. `fn create_package` etc.

I'm thinking that generate sounds like we're doing a bunch of code generation, when really we're mainly taking things that already exist and packaging them together.

Reply:
create sounds good!